### PR TITLE
Add companies

### PR DESCRIPTION
**Pull request by pansay**
|form factor|before|after|
|---|---|---|
|small (iphonex)|||
|medium (ipadpro)|||
|large (desktop)|||

### CODE REVIEW

**Comment by muratogat** [comment]
I can't merge, because the new token images would look bad in the app.
- Please revert the AKS DAKS AKT DAKT icons to non-transparent versions. Transparent icons not really work, because this would imply a single background color of the circle for icons. The app draws them on a white background by default, which doesn't fit our icons.
- Also please revert trimming whitespace around our icons. The icon is not symmetric when all white space is trimmed. There is a larger tip on top, and when trimmed and put into a circular container, the container and our logo circle have varying margins, which looks awful. This was also something I optimized specifically pixel by pixel to match a circular background.

In short, please set the background to whatever color we want it to be. If a circular container is to be used, crop it in the code. Also please revert the AKS DAKS AKT DAKT icons completely.

Thanks,
Murat
